packages/cozy-sharing/src/SharingBanner/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```

Comment:
This is a brand new file (did not exist in cozy-drive).

Not really an HoC but basically has the same use. It supercharges the SharingBanner with display trigger and localization. We used a simple component here instead of an HoC because it was simple enough to solve the problem at hand. Moreover, it is not supposed to be reused.